packages/php/email-editor/tests/integration/Integrations/Core/Renderer/Blocks/Embed_Test.php

```
@@ -534,6 +534,167 @@ public function test_youtube_embed_falls_back_to_link_when_thumbnail_fails(): vo
+	public function test_youtube_embed_falls_back_to_oembed_thumbnail_for_playlists(): void {
+		$mock_thumbnail_url   = 'https://i.ytimg.com/vi/abc/hqdefault.jpg';
+		$mock_oembed_response = wp_json_encode(
+			array(
+				'type'          => 'video',
+				'thumbnail_url' => $mock_thumbnail_url,
+				'title'         => 'Test Playlist',
+			)
+		);
+
+		// Use pre_http_request filter to intercept oEmbed HTTP calls.
+		$filter_callback = function ( $preempt, $args, $url ) use ( $mock_oembed_response ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$args'. (undefined)

(UnusedFormalParameter)